### DISCUSSION TITLE
Support for Multi Item drag and drop in grid or list

### DISCUSSION THREAD

**Discussion by ankursehdev** [atlassian/pragmatic-drag-and-drop#99] [General]
Hello. I was wondering if this library supports Multi Item Drag and Drop. 

the Idea is to select multiple items and drag at once. please let me know if there are examples of how I can implement that.

**Comment by thomasbritton** [comment]
From what I saw whilst looking into this feature myself is that is not support yet, here is a issue asking for it and they are supposedly looking into adding the feature but there hasn't been communication on it for a some time now.

https://github.com/atlassian/pragmatic-drag-and-drop/issues/26

**Comment by nktnet1** [comment]
Hey @ankursehdev & @thomasbritton,

I found an example of this on the official repository [here](https://github.com/atlassian/pragmatic-drag-and-drop/blob/967082b9c01ad5161ea78867417f7e697fb9ee9d/packages/documentation/examples/board-with-multi-drag.tsx).

I've also extracted the code into a standalone react-vite example in [this repository](https://github.com/nktnet1/pragmatic-dnd-multiselect). A deployed demo of the code is available here:
- https://nktnet1.github.io/pragmatic-dnd-multiselect

**Reply by quolpr** [reply]
Big thanks!
